qoffer-domain/src/main/groovy/life/qbic/portal/portlet/offers/identifier/TomatoId.groovy

```diff
@@ -36,8 +36,17 @@ class TomatoId {
         return new Version(version)
     }
 
+    /**
+     * TODO documentation
+     * @since 1.0.0
+     */
     void increaseVersion() {
         this.version.increaseVersion()
+        //TODO  https://github.com/qbicsoftware/offer-manager-2-portlet/issues/232
+        // query for the latest version first and increase this version
+        // make sure the increased version does not already exist
+        // this should be done somewhere with access to the database
+        // and is part of the business rules
```

Comment:
As this is not really part of this PR I would remove it again and tackle it when working on the other card.
```suggestion
```